src/other/utils.jl

```diff
@@ -178,3 +178,133 @@ function _nt_like_hash(v, h::UInt)
 
     return xor(objectid(Tuple(propertynames(v))), h)
 end
+
+_findall(B) = findall(B)
+
+@inline _blsr(x)= x & (x-1) 
+const _msk64 = ~UInt64(0)
```

Comment:
Use capital letters for constants.

Also this seems to be a bit more readable (but if you feel the `~` is better please revert)
```suggestion
const _MSK64 = typemax(UInt64)
```